health/health.go

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2024 The VeChainThor developers
+
+// Distributed under the GNU Lesser General Public License v3.0 software license, see the accompanying
+// file LICENSE or <https://www.gnu.org/licenses/lgpl-3.0.html>
+
+package health
```

Comment:
This this package need to be at the root level? Seems very small

Reply:
Had two comments that has the potential to put this package into the `api/admin` folder.